plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerBehaviour.java

```diff
@@ -131,9 +145,9 @@ public final void setModulePublishState2(IModule[] module, int state) {
    *
    * @param runnables the path to directories that contain configuration files like appengine-web.xml
    * @param console the stream (Eclipse console) to send development server process output to
-   * @param host the host name to which application modules should bind
+   * @param port the app service port
```

Comment:
I think this is not the proper description for the port. Maybe 'server listener port'? @elharo